probability_setup.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+"""
+Script for manual enroll students in edx-adapt application.
+
+Student's anonymous ids are downloaded from Edx lms Course/Instructor/Data Download/ page as csv file.
+Sample of such file is located in edx-adapt/data dir.
+
+Default values for optional parameters --host, --port, --course are taken from config.py file. If script is used
+locally without config.py file nearby, optional parameters: --host, --port, --course become mandatory.
+Additionally could be set optional parameters --prob and --skills if default values are not appropriate.
```

Comment:
could be set optional parameters --prob and --skills -> optional parameters --prob and --skills can be set.
appropriate -> suitable.